lxc/monitor.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/spf13/cobra"
 	"gopkg.in/yaml.v2"
 
+	lxd "github.com/lxc/lxd/client"
```

Comment:
The `lxd ` part isn't needed as `lxd` is its package name, so this should be imported as `"github.com/lxc/lxd/client"`.

Reply:
Gotcha :+1: